scripts/rnnlm/choose_features.py

```diff
@@ -84,9 +84,9 @@
 #  and 'wordlist' is a list indexed by integer id, that returns the string-valued word.
 def read_vocab(vocab_file):
     vocab = {}
-    with open(vocab_file, 'r', encoding="utf-8") as f:
+    with open(vocab_file, 'r', encoding="latin-1") as f:
         for line in f:
-            fields = line.split()
+            fields = line.split(' ')
```

Comment:
@vesis84, you can remove the ' ' here and elsewhere.  If the file contained tabs, we do want to split on those as well, even though that's now how we expect people will write the file.

Reply:
Hi, we could eventually support only the spaces and tabs and nothing else by: 
`re.split("[ \t]", str)` Does that sound good? (it's more explicit and readable...)
K.

Reply:
I'd prefer just split(), I think, it's easier to remember and duplicate.

Reply:
Well, `split()` splits on any whitespace character (this can be a problem for example with `\r` characters). Are you sure you want to allow to split any "whitespace" character? (and sorry for explaining if you already are aware of it...)